spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/data/aerospike/AerospikeDataAutoConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.data.aerospike;
+
+import com.aerospike.client.AerospikeClient;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.aerospike.repository.AerospikeRepository;
+
+/**
+ * {@link EnableAutoConfiguration Auto-configuration} for Spring Data's Aerospike support.
+ *
+ * @author Igor Ermolenko
+ */
+@Configuration
+@ConditionalOnClass({AerospikeClient.class, AerospikeRepository.class})
+@ConditionalOnProperty(prefix = "spring.data.aerospike", value = "host")
```

Comment:
Why `host`? It is recommended to use a list of nodes, so that client could connect to one of the available pods in case one of them is nonoperative.

Reply:
Reimplemented. Properties **host** and **port** replaced with property **hosts**.